### PR TITLE
Swift: local dataflow

### PR DESCRIPTION
**Pull request by rdmarsh2**

### CODE REVIEW

File: swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll [hunk 1]
```diff
@@ -173,7 +221,9 @@ class Unit extends TUnit {
  */
 predicate isUnreachableInCall(Node n, DataFlowCall call) { none() }
 
-newtype LambdaCallKind = TODO_TLambdaCallKind()
+newtype LambdaCallKind =
+  TYieldCallKind() or
+  TLambdaCallKind()
```
Comment (MathiasVP) [inline swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll:226]: I forgot to remove these when I ripped out the guts of the Ruby files. Do you mind replacing this with `TODO_TLambdaCallKind()` or similar again?

**Review by MathiasVP — APPROVED** [review]
LGTM! Only a tiny not-at-all-blocking comment.

File: swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll (hunk 1 above)
Comment (rdmarsh2) [reply swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll:226]: I'll do that as a separate PR

Comment (rdmarsh2) [reply swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll:226]: Actually just saw there's an autoformat failure so I'll fix it in this

**Review by MathiasVP — APPROVED** [review]

**Pull request merged**